lib/scalarm/database/core/mongo_active_record_utils.rb

```diff
@@ -41,6 +41,16 @@ def parse_json_if_string(attribute)
           end
         end
 
+        def create_index(attribute)
```

Comment:
@dkrol - czy nie można byłoby zrobić metody, którą wywołuje się w ten sposób:
```
create_index :user_id, :cloud_name
```
?
Moim zdaniem ułatwia to pisanie[1], kod jest bardziej zwięzły i nawet łatwiej go zrozumieć[1]. Teraz nie ma do tej metody dokumentacji, a widać dwa sposoby użycia - metoda przyjmuje id atrybutu jako String (czy może być też Symbol?) lub Hasha, którego kluczami są nazwy atrybutów, a wartościami ``1``. Jeśli metoda przyjmowała by jeden lub wiele Stringów/Symboli to bardziej pasowałoby do stylu pisania w Rubym[1].

[1] - takie coś jest analogiczneg np. do metody ``attr_accessor`` gdzie można podać wiele Symboli lub Stringów po przecinku